_posts/2024-05-26-update.md

```diff
@@ -29,8 +29,42 @@ CBOR support was proposed by [Ben Luddy in 2019](https://github.com/kubernetes/e
 
 
 ## Other Merges
+* [Enhanced Validation for List Parameters in client-go](https://github.com/kubernetes/kubernetes/pull/125166)
+
+* [Enhanced Progress Notification Timing for Improved Reliability](https://github.com/kubernetes/kubernetes/pull/125115)
+
+* Updated the Kubernetes repository to use [`registry.k8s.io/pause:3.10`](https://github.com/kubernetes/kubernetes/pull/125112)
+
+* [Added -v Flag to Windows Pause Binary](https://github.com/kubernetes/kubernetes/pull/125067 ) Promotes 
+  Consistency Across Platforms.
+
+* Added [`Set` Method](https://github.com/kubernetes/kubernetes/pull/125052 ) to Envvar Implementation in 
+  client-go.
+
+* [Deprecated the RootlessControlPlane feature gate](https://github.com/kubernetes/kubernetes/pull/124997 ) in Kubedam.
+
+* [The `horizontal-pod-autoscaler-upscale-delay` and `horizontal-pod-autoscaler-downscale-delay` flags have been removed](https://github.com/kubernetes/kubernetes/pull/124948)
+
+* [All waitingPods in scheduler now can be obtained from any profiles](https://github.com/kubernetes/kubernetes/pull/124926)
+
+* [CEL default compatibility environment version to updated to 1.30](https://github.com/kubernetes/kubernetes/pull/124779)
+
+* [Improved CBOR Decoding](https://github.com/kubernetes/kubernetes/pull/124775 ) for Enhanced UnstructuredList Consistency in Kubernetes.
+
+* Client-go/reflector: [warns on the missing bookmark events](https://github.com/kubernetes/kubernetes/pull/124614)
+* [Fixed Azure Disk PV Topology Label Issues](https://github.com/kubernetes/kubernetes/pull/124528 )with External Cloud Provider.
+
+* kubeadm [Updated to Use etcd 3.5.11's](https://github.com/kubernetes/kubernetes/pull/124465) New Liveness and Readiness Endpoints
+
+* Improved [Generalization of `checkWatchListConsistencyIfRequested`](https://github.com/kubernetes/kubernetes/pull/124446 ) in client-go.
+
+* [ Container restart due to pod spec field changes has been fixed.](https://github.com/kubernetes/kubernetes/pull/124220)
```

Comment:
right this as what it does, rather than "has been fixed".  "Pod spec field changes will not trigger container restart"